<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>animals_screen</class>
 <widget class="QMainWindow" name="animals_screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Animals</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="delete_animal_button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Animal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toggle_lang_button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Toggle English/German</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_animal_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Animal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edit_animal_button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Animal</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>561</width>
      <height>371</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date Of Birth</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Species</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Breed</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="home_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
